setup.py

```diff
@@ -16,10 +16,10 @@
     ##
     "PyYAML>=3.12",
     "awacs>=0.6.0",
-    "gitpython~=2.0",
-    "schematics~=2.0.1",
+    "gitpython>=2.0,<=3.0",
```

Comment:
I believe you should be using `<` not `<=` - basically `~=2.0`  meants it has to be at least version 2.0, but can't move into 3.0.  Can you update, then I'll merge?  Thanks.

Reply:
Hi,

I have updated the PR.

The first time i pushed the CircleCi tests all failed with hat error. After i rebased on master and pushed again it passed. It may have been a temporary issue :-)